src/main/integrations/screenshots.ts

```diff
@@ -2,21 +2,13 @@ import { defineIntegration } from '@sentry/core';
 import { logger } from '@sentry/utils';
 import { BrowserWindow } from 'electron';
 
-import { ElectronMainOptions } from '../sdk';
-
 /**
  * Captures and attaches screenshots to events
  */
 export const screenshotsIntegration = defineIntegration(() => {
   return {
     name: 'Screenshots',
-    async processEvent(event, hint, client) {
-      const attachScreenshot = !!(client.getOptions() as ElectronMainOptions).attachScreenshot;
-
```

Comment:
I think we should still gate this here behind options in case people dynamically add integrations.

Reply:
Ok, will add these back!